pom.xml

```diff
@@ -110,12 +110,33 @@
         <artifactId>maven-enforcer-plugin</artifactId>
         <executions>
           <execution>
-            <id>enforce-maven-version</id>
+            <id>default-cli</id>
```

Comment:
This name is required to make `mvn enforcer:enforce` work, otherwise it works also via `mvn validate`.